main.js

```diff
@@ -2,43 +2,64 @@ require("dotenv").config();
 
 const fs = require("fs");
 const Discord = require("discord.js");
-let prefix='-';
+const prefix = "-";
 const client = new Discord.Client();
+const commandFiles = fs.readdirSync("./commands").filter((file) => file.endsWith(".js"));
+
 client.commands = new Discord.Collection();
-const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'));
+
+const db = require("./database");
+const { Octokit } = require("@octokit/core");
+const { restEndpointMethods } = require("@octokit/plugin-rest-endpoint-methods");
+const MyOctokit = Octokit.plugin(restEndpointMethods);
 
 for (const file of commandFiles) {
-	const command = require(`./commands/${file}`);
-	client.commands.set(command.name, command);
+  const command = require(`./commands/${file}`);
+  client.commands.set(command.name, command);
 }
 
 client.once("ready", () => {
-    console.log("GitCord Bot is online");
+  console.log("GitCord Bot is online");
 });
+
 // taking in the commands
 client.on("message", (message) => {
+  if (!message.content.startsWith(prefix) || message.author.bot) return;
 
-    if (!message.content.startsWith(prefix) || message.author.bot) return;
+  const args = message.content.slice(prefix.length).split(" ");
+  const command = args.shift().toLowerCase();
 
-    const args = message.content.slice(prefix.length).split(' ');
-    const command = args.shift().toLowerCase();
+  // Message and Command logging
+  console.log("Message is:", message.content);
+  console.log("Command is:", command);
 
-    // Message and Command logging
-    console.log("Message is:" + message.content);
-    console.log("Command is:" + command);
+  if (!client.commands.has(command)) {
+    console.log("Command does not exist.");
+    return;
+  }
 
-    if (!client.commands.has(command)) {
-        console.log("Command does not exist in github.js");
-        return;
+  if (command !== "lighthouse" && command !== "tech-stack" && command !== "help" && command !== "github-info" && command !== 'github-delete-token') {
```

Comment:
I'm just speculating, but you could potentially rewrite this as:

```javascript
const knownCommands = ["lighthouse", "tech-stack", "help", "github-info", "github-delete-token"];
if (!knownCommands.has(command)) {
    // ...
}
```

...which is slightly neater than having to do `&& command !== <thing>` several times over.